CHANGELOG.md

```diff
@@ -9,7 +9,9 @@ and this project adheres to Rust's notion of
 
 ## [0.2.0] - 2022-06-24
 ### Added
+- `orchard::value::ValueSum` now implements `core::ops::Sub`
```

Comment:
Undo this addition (and squash the commits to clean up the branch).

Reply:
Yes, pushed the PR addressing @nuttycom's comments, and then saw yours.